src/richie/apps/courses/defaults.py

```diff
@@ -382,3 +386,16 @@
 # Maximum number of archived course runs displayed by default on course detail page.
 # The additional runs can be viewed by clicking on `View more` link.
 RICHIE_MAX_ARCHIVED_COURSE_RUNS = 10
+
+# Define possible hover color that can be choosen for an MainMenuEntry and to apply on
+# its menu item
+MENU_ENTRY_COLOR_CLASSES = getattr(
+    settings,
+    "RICHIE_MENU_ENTRY_COLOR_CLASSES",
+    # (("", "None"),)
+    (
+        ("", "None"),
+        ("primary", "Primary"),
+        ("warning", "Warning"),
```

Comment:
Those labals should be internationalized.

Reply:
I've only make it for the 'None' item since the other ones were just for local development, i've removed them so there won't be confusion again.